src/sys/aio.rs

```diff
@@ -19,9 +19,13 @@ libc_enum! {
         /// do it like `fsync`
         O_SYNC,
         /// on supported operating systems only, do it like `fdatasync`
-        #[cfg(any(target_os = "openbsd", target_os = "bitrig",
-                target_os = "netbsd", target_os = "macos", target_os = "ios",
-                target_os = "linux"))]
+        #[cfg(any(
```

Comment:
Can you put the first target OS on the same line as `#[cfg(any(`? This is consistent with our styling elsewhere.